k8s/karpenter/provisioners/runners/x86_64/v2/provisioners.yaml

```
@@ -0,0 +1,55 @@
+---
+# Provisioner for x86_64_v2 gitlab runners
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: default
+spec:
+  providerRef:
+    name: default
+
+  # Consolidation will de-provision larger than necessary nodes
+  consolidation:
+    enabled: true
+
+  # Resource limits for this provisioner only
+  limits:
+    resources:
+      cpu: 5k
+      memory: 20Ti
+
+    # Only spin up amd64 nodes
```

I think the `requirements` section got dropped here

```suggestion

    requirements:
    # Only spin up amd64 nodes
```